### PR TITLE
Documentation: behaviour with empty LDAP user-attributes undocumented…

### PR DESCRIPTION
**Pull request by fgsl**
… #1010

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @blizzz

File: admin_manual/configuration_user/user_auth_ldap.rst [hunk 1]
```diff
@@ -735,3 +735,10 @@ is offline and will not try to connect again for the time specified in **Cache
 Time-To-Live**. If you have a backup server configured Nextcloud will connect to
 it instead. When you have scheduled downtime, check **Disable Main Server**  to
 avoid unnecessary connection attempts.
+
+Note
+----
+
+When a LDAP object's name or surname is left empty, Nextcloud will treat it as an
```
Comment (blizzz) [inline admin_manual/configuration_user/user_auth_ldap.rst]: To be more precise, it is about the (configured) display name attribute, by default "displayname".

Comment (fgsl) [reply admin_manual/configuration_user/user_auth_ldap.rst]: I changed the text, to be more precise.

**Pull request merged**